test/e2e/azure_lb.go

```diff
@@ -267,7 +274,12 @@ func AzureLBSpec(ctx context.Context, inputGetter func() AzureLBSpecInput) {
 			return err
 		}
 		return nil
-	}, deleteOperationTimeout, retryableOperationSleepBetweenRetries).Should(Succeed())
+	}, retryableOperationTimeout, retryableOperationSleepBetweenRetries).Should(Succeed())
+	Logf("waiting for the external LB service to be deleted: %s", elbService.Name)
```

Comment:
did this to be consistent but need to check if it adds a lot of time to e2e duration since we don't strictly need to wait on this one (it's the last service to be reconciled)

Reply:
Interesting, so we have actually not been validating ELB delete (ILBs and ELBs are ultimately the same Azure LoadBalancer type, so not super critical, but I agree it's best-practice to not make an exception here)